small-talk/app/homepage/page.js

```diff
@@ -1,29 +1,14 @@
 "use client"
 import Link from 'next/link'
-import React, { useState } from 'react';
-import SideBar from '../components/SideBar';
-import TopBar from '../components/TopBar';
-
-const handleRedirect = (path) => () => {
-    window.location.href = path;
-}
+import React from 'react';
+import ThemeLayout from '../components/ThemeLayout';
 
 export default function Homepage() {
     return (
-        <div className="homePageContainer">
-            {/* header for homepage */}
-            <h1 className="header">Welcome to Homepage</h1>
+        <ThemeLayout>
             <div>
-                <TopBar/>
-            </div>
-            <div>
-                <SideBar/>
-                <div>
-
-                </div>
+                <h1>Homepage</h1>
             </div>
-            {/* <img src="/img/agenda.jpeg" altText="Agenda" style={{ height: '175px', width: '1000px', marginTop: '150px', marginLeft: '500px' }}/>
-            <img src="/img/calendar.png" altText="Calendar" style={{ width: '1000px', height: '500px', marginTop: '30px', marginLeft: '500px' }}/> */}
-        </div>
+        </ThemeLayout>
```

Comment:
Feels good to see this cut down so much

Reply:
That is very nice to see it come down a lot. Adding the reusable components is really useful for uniformity and simplicity for the code so far.